parent/pom.xml

```diff
@@ -287,6 +287,12 @@
                 <version>${activemq-version}</version>
                 <scope>test</scope>
             </dependency>
+            <dependency>
```

Comment:
If this is used only in the tests for sums it makes sense to add the dependency only in the specific test folder and not in parent